src/retinanet.py

```diff
@@ -5,6 +5,8 @@
 from torchvision import models
 from math import sqrt
 
+from .steppy.base import BaseTransformer
```

Comment:
@apyskir please import base transformer from the steppy library directly -> `from steppy.base import BaseTransformer`

Reply:
@jakubczakon OK, right.